src/demos/calcite-graph.html

```diff
@@ -42,9 +52,22 @@ <h3>Inherit Color</h3>
         [90, 10],
         [100, 0]
       ];
-      [].slice.call(document.querySelectorAll("calcite-graph")).forEach(function (graphElement) {
+      document.querySelectorAll("calcite-graph").forEach(function (graphElement) {
         graphElement.data = data;
       });
+
+      // Add highlight ranges
+      document.querySelectorAll(".highlight-range").forEach(function (graphElement) {
+        graphElement.highlightMin = 25;
+        graphElement.highlightMax = 75;
+      });
+
+      // Add color stops
+      var rainbow = ["red", "orange", "yellow", "green", "cyan", "blue", "violet"];
+      var colorStops = rainbow.map((color, i) => [(1 / (rainbow.length - 1)) * i, color]);
+      document.querySelectorAll(".color-stops").forEach(function (graphElement) {
```

Comment:
Would it be possible to set the color stops in the [histogram slider use case](https://esri.github.io/calcite-components/?path=/story/components-controls-slider--histogram)? From what I understand, it doesn't seem like it. Am I missing something?

Reply:
I split the the slider functionality off into another PR (based on this one) https://github.com/Esri/calcite-components/pull/2573.

If you would prefer them to be in the same PR I can do that.

Reply:
I moved the changes from #2573 here and closed it.